LiteLoader/include/llapi/EventAPI.h

```diff
@@ -285,7 +282,7 @@ class PlayerAttackEvent : public EventTemplate<PlayerAttackEvent> {
 public:
     Player* mPlayer = nullptr;
     Actor* mTarget = nullptr;
-    int mAttackDamage = false;
+    float mAttackDamage = 0;
```

Comment:
maybe we need some tricks to keep ABI stability here

Reply:
Can we just undo this changes?  It seems that the damage values ​​are all integers.